src/fr/guides/virtual-meetings/index.html

```diff
@@ -0,0 +1,313 @@
+---
+title: Pratiques exemplaires de tenue de réunions virtuelles accessibles
+layout: layouts/base.njk
+description: Ceci est un guide pour la tenue de réunions virtuelles accessibles créé par Accessibilité, stratégie et planification / Bureau du dirigeant principal de l’information Services partagés Canada.
+documents:
+fileTitle: "Pratiques exemplaires de tenue de réunions virtuelles accessibles"
+filename: "/docs/alternate-formats/PratiquesExemplairesPourLaTenueDeRéunionsVirtuellesAccessibles.docx"
+version: 2
+size: 93
+sizeFormat: ko
+---
+
+<h2 class="mrgn-bttm-lg">Autres formats du document (à télécharger)</h2>
+<ul class="btn-toolbar list-inline" xmlns="http://www.w3.org/1999/html">
+	<li class="btn-group mrgn-bttm-md">
+		<a href="/docs/alternate-formats/PratiquesExemplairesPourLaTenueDeRéunionsVirtuellesAccessibles.docx" class="btn btn-primary"><i class="fas fa-file-word" aria-hidden="true"></i>{{title}} (Word version 2, 93 <abbr title="kilo-octet" aria-label="kilo-octet">ko</abbr>)</a>
```

Comment:
Please remove `aria-label="kilo-octet"` using both `title` and `aria-label` can cause a reader to repeat the values.

Reply:
done